src/services/ui/src/pages/form/chip-form.tsx

```diff
@@ -0,0 +1,235 @@
+import { z } from "zod";
+import { Link, useLocation } from "react-router-dom";
+import { useGetUser } from "@/api/useGetUser";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { submit } from "@/api/submissionService";
+import { zodResolver } from "@hookform/resolvers/zod";
+import {
+  Alert,
+  BreadCrumbs,
+  LoadingSpinner,
+  SectionCard,
+  SimplePageContainer,
+} from "@/components";
+import { optionCrumbsFromPath } from "@/pages/create/create-breadcrumbs";
+import * as Inputs from "@/components/Inputs";
+import { FAQ_TARGET } from "@/routes";
+import { Authority } from "shared-types";
+import {
+  zAttachmentOptional,
+  zAttachmentRequired,
+  zSpaIdSchema,
+} from "@/pages/form/zod";
+import * as Content from "@/pages/form/content";
+import { ModalProvider, useModalContext } from "@/pages/form/modals";
+import { formCrumbsFromPath } from "@/pages/form/form-breadcrumbs";
+
+const formSchema = z.object({
+  id: zSpaIdSchema,
+  additionalInformation: z.string().max(4000).optional(),
+  attachments: z.object({
+    currentStatePlan: zAttachmentRequired({ min: 1 }),
+    amendedLanguage: zAttachmentRequired({ min: 1 }),
+    coverLetter: zAttachmentRequired({ min: 1 }),
+    budgetDocuments: zAttachmentOptional,
+    publicNotice: zAttachmentOptional,
+    tribalConsultation: zAttachmentOptional,
+    other: zAttachmentOptional,
+  }),
+  proposedEffectiveDate: z.date(),
+});
+type ChipFormSchema = z.infer<typeof formSchema>;
+
+// first argument in the array is the name that will show up in the form submission
+// second argument is used when mapping over for the label
+const attachmentList = [
+  { name: "currentStatePlan", label: "Current State Plan", required: true },
+  {
+    name: "amendedLanguage",
+    label: "Amended State Plan Language",
+    required: true,
+  },
+  {
+    name: "coverLetter",
+    label: "Cover Letter",
+    required: true,
+  },
+  {
+    name: "budgetDocuments",
+    label: "Budget Documents",
+    required: false,
+  },
+  { name: "publicNotice", label: "Public Notice", required: false },
+  { name: "tribalConsultation", label: "Tribal Consultation", required: false },
+  { name: "other", label: "Other", required: false },
+] as const;
+
+export const ChipForm = () => {
+  const location = useLocation();
+  const { data: user } = useGetUser();
+  const { setCancelModalOpen, setSuccessModalOpen } = useModalContext();
+  const handleSubmit: SubmitHandler<ChipFormSchema> = async (formData) => {
```

Comment:
We can offload some work to TS by wrapping handleSubmit within rhf's form.handleSubmit:

```js
  const form = useForm<ChipFormSchema>({
    resolver: zodResolver(formSchema),
  });

  const handleSubmit = form.handleSubmit(async (formData) => {
      // formData is ChipFormSchema
      try {
      await submit<ChipFormSchema>({
        data: formData,
        endpoint: "/submit",
        user,
        authority: Authority.CHIP_SPA,
      });
      setSuccessModalOpen(true);
    } catch (e) {
      console.error(e);
    }
  });
```